plugins/chat/app/jobs/regular/chat/notify_mentioned.rb

```diff
@@ -3,184 +3,100 @@
 module Jobs
```

Comment:
Ideally I would prefer Jobs to be almost logicless, I see them as controllers. We should maybe consider moving all this code to services.